### ISSUE TITLE
Changing styles (especially fonts) for slides

### ISSUE THREAD

**Issue by felix-wo** [eerotal/LibreSignage#28]
What is the recommended way to change the styles (especially the fonts, in the first place) of the slides to match a corporate design?

**Comment by eerotal** [comment]
* Text color can be changed using the `color´ tag.
* Headings and lead paragraphs can be created with the `h` and ´lead´ tags respectively.
* Italic and bold text can be created with the ´i´ and ´b´ tags respectively.

These tags are described in the markup documentation in the *Help* page. Unfortunately, I have completely overlooked changing fonts. That's an important feature, however, so I'll code something and probably include it in the next patch release, even though patches aren't usually meant for new features.

**Comment by felix-wo** [comment]
Works, and I like it. Thanks!

But let me make a further suggestion: In a consistent (corporate) design i need a specific and fixed font for paragraph text and maybe another for headlines. Maybe there’s even a need to style bold text in a specific way. There should be a possibility to style these classes on a general level. So the CSS should be adjustable.

- Maybe you could implement a settings sub page for basic font settings? This could be per queue, as different slide shows might be independent from another, but global default settings for all content (all queues) seem useful to me, too.
- For more sophisticated tweaking you could perhaps implement the possibility to add a user defined .css files (again globally or per queue). They should not get overwritten with updates of LibreSignage.
- Maybe both.

Detail: Web fonts should be supported.

**Comment by eerotal** [comment]
That's actually a very good idea. How about this:

*  Create a separate settings sub page where admins or possibly normal users could create CSS stylesheets available to all users.
* Add a dropdown to the editor page where users could select one of the existing stylesheets to add a uniform style to the slide.

I think this would be quite flexible since a queue could either have a uniform styling in all slides or different styles in different slides.

**Comment by felix-wo** [comment]
Well, something like this, at least as a next step of development. That would be great!

I think, a default CSS for a whole queue would make sense. Maybe a default plus a choice per slide as well?

**Comment by eerotal** [comment]
Yeah, I was thinking something like that too. So by default select the global style of the whole queue and then the user can change it if they wish to do that. This will have to be implemented in 0.3.0 though so it will take some time. I have some other rather big under the hood changes I'm working on right now.

**Comment by felix-wo** [comment]
Sounds good!
